### PR TITLE
Add ansiblerole-satellite-receptor-installer

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
the role is finally tagged and released, I tried to package myself but I'm missing something obivous, annex can't find the file

```
[2020-03-10T06:43:45.521Z] fatal: [ansiblerole-satellite-receptor-installer]: FAILED! => changed=false 
[2020-03-10T06:43:45.521Z]   cmd: git annex whereis /home/jenkins/workspace/foreman-packaging-rpm-pr-test/packages/plugins//ansiblerole-satellite-receptor-installer/satellite_receptor_installer-0.5.0.tar.gz 2>/dev/null | grep -q "web:"
[2020-03-10T06:43:45.521Z]   delta: '0:00:00.265470'
[2020-03-10T06:43:45.521Z]   end: '2020-03-10 06:43:45.411299'
[2020-03-10T06:43:45.521Z]   msg: non-zero return code
[2020-03-10T06:43:45.521Z]   rc: 1
[2020-03-10T06:43:45.521Z]   start: '2020-03-10 06:43:45.145829'
[2020-03-10T06:43:45.521Z]   stderr: ''
[2020-03-10T06:43:45.521Z]   stderr_lines: <omitted>
[2020-03-10T06:43:45.521Z]   stdout: ''
[2020-03-10T06:43:45.521Z]   stdout_lines: <omitted>
```

File: rel-eng/tito.props [hunk 1]
```diff
@@ -499,6 +499,7 @@ whitelist = nodejs-babel-plugin-module-resolver
 disttag = .el7
 whitelist = ansiblerole-foreman_scap_client
   ansiblerole-insights-client
+  ansiblerole-satellite-receptor-plugin
```
Comment (evgeni) [inline rel-eng/tito.props]:
```suggestion
  ansiblerole-satellite-receptor-installer
```

The error you pasted is not the error you're looking for.

The real problem is that you did whitelist the wrong package name (why do we have so many names anyways?).

**Comment by evgeni** [comment]
@ares it built fine now after changing the package name in tito.props :)

File: packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+%global role_orgname %{repo_orgname}
+%global role_name satellite_receptor_installer
+
+Name: ansiblerole-satellite-receptor-installer
+Summary: Packaging of the satellite_receptor_installer Ansible role
+Version: 0.5.0
+Release: 1%{?dist}
+License: GPLv3
+
+Source0: https://github.com/%{repo_orgname}/%{repo_name}/archive/v%{version}.tar.gz#/%{role_name}-%{version}.tar.gz
```
Comment (evgeni) [inline packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec]: I'd *prefer* if the source would be the tarball on galaxy, but the role does not seem to be uploaded to galaxy yet. (not a blocker)

File: packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Url: https://github.com/%{repo_orgname}/%{repo_name}/
+BuildArch: noarch
+
+Requires: ansible
+
+%description
+This package installs the satellite_receptor_installer Ansibile role.
```
Comment (evgeni) [inline packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec]: It'd be nice to know what the role does ;)

File: packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec [hunk 4]
```diff
@@ -0,0 +1,40 @@
+%global repo_orgname project-receptor
+%global repo_name satellite-receptor-installer
```
Comment (evgeni) [inline packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec]: not strictly your playground (unless you are also upstream?) but I find this repo name and role name difference odd.

File: packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec (hunk 2 above)
Comment (ares) [reply packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec]: it is [here](https://galaxy.ansible.com/project-receptor/satellite_receptor_installer), I just followed what I've seen in other ansible role specs, happy to change it

File: packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec (hunk 3 above)
Comment (ares) [reply packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec]: ok, will add a bit more description of the role

File: packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec (hunk 4 above)
Comment (ares) [reply packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec]: meaning dashes vs underscores? not my choice, I have to believe that's best practices

File: packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec (hunk 2 above)
Comment (evgeni) [reply packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec]: FTR, galaxy does not host roles, just collections, so my comment is moot.

**Pull request merged**